Makefile

```diff
@@ -89,15 +89,6 @@ $(BUILTIN_ACTORS_BUNDLE): $(BUILTIN_ACTORS_CODE)
 	rustup target add wasm32-unknown-unknown && \
 	cargo run --release -- -o output/$(shell basename $@)
 
-
-# Compile the ABI artifacts and Rust bindings for the IPC Solidity actors.
-ipc-actors-abi: $(IPC_ACTORS_ABI)
```

Comment:
I think you could leave this in and only remove the `cargo build ...` stuff, similarly to how the `actor-bundle` target works. The reason is that it's still needed for the docker-build and it's just a more convenient command to have, in case it's needed.